client/src/components/globals/Signup.jsx

```diff
@@ -67,7 +70,17 @@ export default function Signup() {
 
           .catch((err) => setError(err.code));
       }
-      navigate("/login");
+      toast.success("Verification link sent to email", {
```

Comment:
Is it being sent automatically using firebase? Or do we make some API call? - If yes, then we should have try-catch for error instead of assuming that it sent email always.

Reply:
Yes , It is send using firebase , firebase has a sendVerificationEmail function for this purpose